build-scan-shared/src/setup/input.ts

```diff
@@ -53,7 +62,11 @@ export function isAddProjectIdInJobSummary(): boolean {
     return getBooleanInput('add-project-id-in-job-summary')
 }
 
-export function exportVariables(buildTool: commonBuildTool.BuildTool): void {
+export function exportVariables(accessToken: string, buildTool: commonBuildTool.BuildTool): void {
+    if (accessToken) {
+        core.setSecret(accessToken)
+        core.exportVariable(auth.ENV_KEY_DEVELOCITY_ACCESS_KEY, accessToken)
```

Comment:
Do you need to set the deprecated env var here?

Reply:
This action is compatible with the Develocity extension only.
We decided not to provide support for the GE extension because the first known consumer had already migrated to DV when starting using it.